composer/utils/entrypoint.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for defining custom entrypoints."""
+
+from typing import Type
+
+__all__ = ['_warning_on_one_line']
```

Comment:
nit: private methods should not be exported.

Would it make sense to call this file as `warning_helpers.py`? idk, don't have strong opinions on the name.

Reply:
yup renamed!